build.gradle

```diff
@@ -29,29 +30,54 @@ targetCompatibility = 11
 ext {
 	depVersion = [
 		commonsJava              : "2.3.5",
+		commonsLang3             : "3.7",
 		confuse                  : "1.2.1",
 		confuseIoYaml            : "1.0.0",
 		disruptor                : "3.4.2",
 		dockerJava               : "3.0.14",
-		grpc                     : "1.8.0",
-		guava                    : "20.0",
+		grpc                     : "1.17.1",
+		guava                    : "27.0.1-jre",
 		log4j                    : "2.8.2",
 		mongooseBase             : "4.2.8",
 		mongooseStorageDriverCoop: "4.2.17",
-		opencensus               : "0.8.0",
-		pravega                  : "0.4.0",
-		protobuf                 : "3.4.0",
+		netty                    : "4.1.16.Final",
+		nettyTcNative            : "2.0.17.Final",
```

Comment:
Why do we depend on netty now and not pravega?

Reply:
because the dependencies are the jar files and not maven artifacts
the dependencies of the built pravega jar files should be resolved "manually"